MapboxNavigation/Style.swift

```diff
@@ -94,7 +100,14 @@ open class FloatingButton: Button {
         }
     }
     
-    class func rounded<T: FloatingButton>(image: UIImage? = nil, selectedImage: UIImage? = nil, size: CGSize = FloatingButton.buttonSize) -> T {
+    /**
+     Return a `FloatingButton` with given images and size.
+     
+     - parameter image: The `UIImage` of this button.
+     - parameter selectedImage: The `UIImage` of this button when selected.
+     - parameter size: The size of this button, is default as the `FloatingButton.buttonSize` to keep consistent.
```

Comment:
> The size of this button, or `FloatingButton.buttonSize` if this argument is not specified.

Reply:
Thanks for the suggestion!  I've edited the documentation as suggested.